cmd/scollector/collectors/exim_linux.go

```diff
@@ -0,0 +1,113 @@
+package collectors
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"bufio"
+	"bytes"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_exim_mailq, Interval: time.Minute * 1})
+}
+
+func c_exim_mailq() (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	mailq := exec.Command("/usr/bin/mailq")
```

Comment:
Because I need to pipe the commands
On Jan 15, 2015 1:03 PM, "Matt Jibson" <notifications@github.com> wrote:

> In cmd/scollector/collectors/exim_linux.go
> <https://github.com/bosun-monitor/bosun/pull/658#discussion-diff-23029131>
> :
>
> > +	"time"
> > +
> > +	"bufio"
> > +	"bytes"
> > +
> > +	"bosun.org/metadata"
> > +	"bosun.org/opentsdb"
> > +)
> > +
> > +func init() {
> > +	collectors = append(collectors, &IntervalCollector{F: c_exim_mailq, Interval: time.Minute * 1})
> > +}
> > +
> > +func c_exim_mailq() (opentsdb.MultiDataPoint, error) {
> > +	var md opentsdb.MultiDataPoint
> > +	mailq := exec.Command("/usr/bin/mailq")
>
> Why not use our existing command utils? They do good things like timeout
> if it doesn't complete
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bosun-monitor/bosun/pull/658/files#r23029131>.
>